installer/cli/deploy/standalone/extensions-all-jvm/docker-compose.yml

```diff
@@ -19,10 +19,12 @@ services:
     image: ${SP_DOCKER_REGISTRY}/extensions-all-jvm:${SP_VERSION}
     depends_on:
       - consul
-      - backend
       - couchdb  
     volumes:
       - files:/spImages
+# Set SP_HOST to an IP or hostname that can be resolved from the Docker container and local-running backend
```

Comment:
Should we cleanup this comment?

Reply:
I removed the comment and will add the info to the documentation, thanks!